### DISCUSSION TITLE
Translational equivalent to Mechanics.Rotational.Components.IdealGear

### DISCUSSION THREAD

**Discussion by grondilu** [modelica/ModelicaStandardLibrary#3930] [Ideas]
A [scissor mechanism](https://en.wikipedia.org/wiki/Scissors_mechanism) can be seen as a linear analog to a gear.   Yet as far as I can tell it is not implemented in `Mechanics.Translational.Components`.

It could be called `IdealScissor`.  I may try to implement it but I'm new to Modelica so I would probably do a terrible job.

PS.  Like maybe something like this :

```within Modelica.Mechanics.Translational.Components;
model IdealScissor "Idealized scissor mechanism without inertia"
  extends Modelica.Mechanics.Translational.Interfaces.PartialElementaryTwoFlangesAndSupport2;
  parameter Real ratio(start=1) "Transmission ratio";

equation
  s_a = -ratio*s_b;
end IdealScissor;
```

though that may be too easy.   Would it work?

**Comment by HansOlsson** [comment]
Interesting - but to me a scissor mechanism seem more 3D (or 2D) than 1D-Translational; but I'm no expert in using such models.

**Reply by grondilu** [reply]
It is reduced to 1D in the idealized model where we neglect its mass.

**Comment by christiankral** [comment]
The two translational flanges act in different directions, however. This might be confusing when placed in a 1D translational package. Some more issues are:

- The force balance is not yet considered in the proposed model
- The model would then require two support flanges for either direction

I wonder if there exists a purely 1D scissors model which could be implemented. Otherwise this model rather belongs to 2D or 3D implementation.

**Comment by tobolar** [comment]
I agree with @christiankral .
The suggested solution is oriented on `Rotational.Components.IdealGear`, having two flanges and a support, and could be implemented similarly. But the question is, what is a use case for such an element. 

IMO a scissors, as a real life example, would need four flanges. One of them could be declared as support per default. An implementation is possible.

**Comment by beutlich** [comment]
For the reference, there also is #1873.
